sql/migrations/local/0004_atx_sync.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE atx_sync_state 
+(
+    epoch     INT NOT NULL,
+    id        CHAR(32) NOT NULL,
+    requests  INT NOT NULL DEFAULT 0, 
+    PRIMARY KEY (epoch, id)
+) WITHOUT ROWID;
```

Comment:
`WITHOUT ROWID` may produce overflow pages / slow clustered index in SQLite, given the ID size here

Reply:
afaik we are using default page size of 4096 bytes. 

i made it this way so that i can read all id/requests pairs for an epoch from a single tree. if i do primary key (epoch , id) then i first will scan integers from primary key index, and then get those indexes from table with row id.